src/xdocs/checks/coding/missingnullcaseinswitch.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+  <head>
+    <title>MissingNullCaseInSwitch</title>
+  </head>
+  <body>
+    <section name="MissingNullCaseInSwitch">
+      <p>Since Checkstyle 10.18.0</p>
```

Comment:
Release is tomorrow. You might as well bump this just to be on the safe side.

Reply:
bumped to 10.19.0

Reply:
I think check-since workflow fails because it requires the version to be 10.18.0 after the release it should be green after restarting

Reply:
I am sorry, but due to release issues it is probably now safer to revert this to 10.18.0 . I wrongly assumed release would go without an issue.

Reply:
done